upgrade_guide/README.md

```diff
@@ -1,5 +1,63 @@
 ## Upgrade guide
 
+### 4.17.0
+
+#### Beta support for Java17
+
+With the completion of [JAVA-3042](https://datastax-oss.atlassian.net/browse/JAVA-3042) the driver now passes our
+test suite when run on Java17.  We also now include Java17 in our automated test matrix for Java driver releases.
+All features appear to function normally when run with Java17 but we do not want to declare full support for this
+platform until we've received feedback from other users in the ecosystem.
+
+If you discover an issue with the Java driver running on Java17 please let us know!  Java17 issues will be triaged
+and addressed just like issues from any supported Java version.
```

Comment:
```suggestion
With the completion of [JAVA-3042](https://datastax-oss.atlassian.net/browse/JAVA-3042), the driver now passes our automated test matrix for Java driver releases.
While all features function normally when run with Java 17 tests, we do not offer full support for this
platform until we've received feedback from other users in the ecosystem.

If you discover an issue with the Java driver running on Java 17, please let us know. We will traige and address Java 17 issues.
```

Reply:
Any of the regular reporting mechanisms would work.  Probably the default way would be to contact support but posting tickets to JIRA, posting to the Google group/mailing list, etc.... any of those would be fine.

I didn't specify 'cause I didn't want to create the perception of limiting it to any one mechanism.  I'm open to other ideas here, though.